tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
+
 	private $tmpDir;
 
+	protected function getPort()
+	{
+		return $this->startPort + $this->triedPorts;
+	}
+
+	protected function getIPAddress()
+	{
+		$records = dns_get_record(php_uname('n'), DNS_A);
```

Comment:
Looked up the 0.0.0.0 and ::1. 0.0.0.0 means "all local IPv4 addresses" and "::" is the IPv6 equivalent. So it should be safe to use 0.0.0.0 for IPv4 and :: for IPv6